src/Tests/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildAP2PrefernceWithTargetPlatform.cs

```diff
@@ -0,0 +1,49 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.NET.TestFramework;
+using Microsoft.NET.TestFramework.Assertions;
+using Microsoft.NET.TestFramework.Commands;
+using Xunit;
+using FluentAssertions;
+using Xunit.Abstractions;
+using Microsoft.NET.TestFramework.ProjectConstruction;
+
+namespace Microsoft.NET.Build.Tests
+{
+    public class GivenThatWeWantToBuildAP2PrefernceWithTargetPlatform : SdkTest
```

Comment:
Spelling / capitalization nit:

```suggestion
    public class GivenThatWeWantToBuildAP2PReferenceWithTargetPlatform : SdkTest
```